jupyterlab/labapp.py

```diff
@@ -538,6 +537,14 @@ class LabApp(NotebookConfigShimMixin, LabServerApp):
 
     collaborative = Bool(False, config=True, help="Whether to enable collaborative mode.")
 
+    collab_file_poll_interval = Int(
```

Comment:
How about a more explicit name for this public trait?

```suggestion
    collaborative_file_poll_interval = Int(
```

Reply:
A shorter `file_poll_interval` could also work.